src/components/ClickCatcher/ClickCatcher.js

```diff
@@ -1,41 +1,58 @@
-import PropTypes from 'prop-types'
 import React from 'react'
-import { HYLO_URL_REGEX } from 'util/navigation'
+import { useHistory } from 'react-router-dom'
+import { PathHelpers, HYLO_URL_REGEX } from 'hylo-shared'
 
-export default function ClickCatcher ({ tag, handleMouseOver, navigate, ...props }) {
-  if (!['div', 'span', 'p'].includes(tag)) {
-    throw new Error(`invalid tag for ClickCatcher: ${tag}`)
-  }
+export default function ClickCatcher ({ handleMouseOver, groupSlug, ...props }) {
```

Comment:
`ClickCatcher` has been substantially re-factored inline with the backend HTML normalizing/processing now done entirely in `RichText.js` on the backend. Mention and Topic references in content are `span` tags and not links. They are converted to links in the backend for emails and such, and caught here when clicked. HyloHTML content from our editor (but not necessarily things like group.description, etc) now should always be wrapped in `ClickCatcher`. The same is essentially true in Mobile with a slightly different mechanism for catching the clicks. 

The reason for making them spans vs anchors is that they do not get stored with a path or URL. This is also the native way that TipTap generates these entities. 

This is all FYI, and relevant potentially to external API usage. We may want to have a flag for that case which converts these spans to links on the backend. The code is there to do that already if we end-up needing that.

Reply:
Links that are not local Hylo links will ALWAYS open in `target='_blank'`